src/ucp/core/ucp_context.c

```diff
@@ -234,6 +234,10 @@ static ucs_config_field_t ucp_config_table[] = {
    "another thread, or incoming active messages, but consumes more resources.",
    ucs_offsetof(ucp_config_t, ctx.flush_worker_eps), UCS_CONFIG_TYPE_BOOL},
 
+  {"UNIFIED_MODE", "n",
+   "Enable various optimizations intended for homogeneous environment. \n",
```

Comment:
need better explanation what this means , something like "it's guaranteed that the local transport resources/devices of all entities which connect to each other are the same"